src/main/java/me/clip/placeholderapi/replacer/RegexReplacer.java

```diff
@@ -51,17 +51,15 @@ public String apply(@NotNull final String text, @Nullable final OfflinePlayer pl
     }
 
     final StringBuffer builder = new StringBuffer();
+    PlaceholderExpansion expansion;
+    String requested;
 
     do {
-      final String identifier = matcher.group("identifier");
-      final String parameters = matcher.group("parameters");
-
-      final PlaceholderExpansion expansion = lookup.apply(identifier);
-      if (expansion == null) {
+      if ((expansion = lookup.apply(matcher.group("identifier"))) == null) {
         continue;
       }
 
-      final String requested = expansion.onRequest(player, parameters);
+      requested = expansion.onRequest(player, matcher.group("parameters"));
```

Comment:
Same here

Reply:
The regex replacer isn't even used, so this is completely pointless to "improve". If you want to improve it, remove it completely.